src/Captioning/Format/WebvttFile.php

```diff
@@ -6,98 +6,92 @@
 
 class WebvttFile extends File
 {
-    const TIMECODE_PATTERN = '#^([0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}) --> ([0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{3}) ?(.*)$#';
+    const TIMECODE_PATTERN = '#^((?:[0-9]{2}:)?[0-9]{2}:[0-9]{2}.[0-9]{3}) --> ((?:[0-9]{2}:)?[0-9]{2}:[0-9]{2}.[0-9]{3})( .*)?$#';
```

Comment:
Corrected to right match patterns like in https://w3c.github.io/webvtt/#introduction example.